Source/JavaScriptCore/dfg/DFGJITCode.cpp

```diff
@@ -35,6 +35,29 @@
 
 namespace JSC { namespace DFG {
 
+JITData::JITData(const JITCode& jitCode, ExitVector&& exits)
+    : Base(jitCode.m_linkerIR.size())
+    , m_stubInfos(jitCode.m_unlinkedStubInfos.size())
+    , m_exits(WTFMove(exits))
+{
+    for (unsigned i = 0; i < jitCode.m_linkerIR.size(); ++i) {
+        auto entry = jitCode.m_linkerIR.at(i);
+        switch (entry.type()) {
+        case LinkerIR::Type::StructureStubInfo: {
+            unsigned index = bitwise_cast<uintptr_t>(entry.pointer());
+            const UnlinkedStructureStubInfo& unlinkedStubInfo = jitCode.m_unlinkedStubInfos[index];
+            StructureStubInfo& stubInfo = m_stubInfos[index];
+            stubInfo.initializeFromDFGUnlinkedStructureStubInfo(unlinkedStubInfo);
+            at(i) = &stubInfo;
+            break;
+        }
+        default:
+            at(i) = entry.pointer();
```

Comment:
Yeah, I think we can eventually make this more solid, but I think it is OK for now because I would make that assertion relaxed or more complicated, and they are based on the design of how we will implement next things in uDFG.